gogs_client/entities.py

```diff
@@ -240,3 +240,34 @@ def pull(self):
             :rtype: bool
             """
             return self._pull
+
+
+class GogsKey(object):
+
```

Comment:
Could you add a docstring for the `GogsKey` class, and specify that keys represent SSH keys?